rts.tests/supervise-base.sh

```diff
@@ -5,10 +5,17 @@
 # svscanboot
 
 echo '--- supervise starts, svok works, svup works, svstat works, svc -x works'
+rm -rf test.sv
+mkdir test.sv
+catexe test.sv/run <<EOF
+#!/bin/sh
+echo hi
+EOF
+touch test.sv/down
 supervise test.sv &
 until svok test.sv
 do
-  sleep 1
+  sleep 0
```

Comment:
I'm puzzled. What's the point of sleep 0?

Reply:
It's faster than sleep 1. =) Seriously, I didn't think fractional sleep times, or usleep/nanosleep, etc would be portable. So, short of adding a proprietary sleep in the distro, the only other idea I had to waste a bit of time was something like "echo foo >/dev/null".

Reply:
On second thought, what about something like ```sleeper -n -d 10000000``` for a 100 ms sleep instead? At this point, adding another switch would be trivial.